src/engine/Filter.cpp

```diff
@@ -657,3 +676,27 @@ void Filter::computeResultFixedValue(
   LOG(DEBUG) << "Filter result computation done." << endl;
   resultTable->_data = result.moveToDynamic();
 }
+
+std::string Filter::stringToUri(const std::string& s) {
+  size_t r = s.rfind('"');
+  return "<" + s.substr(1, r - 1) + ">";
+}
+
+std::string Filter::stringRemoveTrailingQuotationMark(const std::string& s) {
```

Comment:
I can't seem to find any call sites for these? Did you intent to use these functions in the hack above but then decided not to?

Reply:
I was originally trying to properly implement the handling of uris and strings and wrote those funcions for that, but then realized that a proper implementation would require more significant changes. I've deleted them now.